platforms/hyperledger-fabric/configuration/roles/create/chaincode/invoke/tasks/valuefile.yaml

```diff
@@ -2,20 +2,50 @@
 ############################################################################################
 # This role creates the invoke_chaincode value file
 
-# Check or Wait if install-chaincode is already run
-- name: "Waiting for chaincode to be installed on {{ peer.name }}"
+# Check or Wait if commit-chaincode is already run
+- name: "Waiting for chaincode to be commited on {{ peer.name }}"
   include_role:
     name: "{{ playbook_dir }}/../../shared/configuration/roles/check/helm_component"
   vars:
     component_type: "Job"
     namespace: "{{ org.name |lower }}-net"
     kubernetes: "{{ org.k8s }}"
-    component_name: "installchaincode-{{ peer.name }}-{{ peer.chaincode.name }}-{{ peer.chaincode.version }}"
+    component_name: "commitchaincode-{{ peer.chaincode.name }}-{{ peer.chaincode.version }}"
```

Comment:
The fabric v1.4.x is not supporting a `commit-chaincode job`, so In case of fabric v1.4.x it should be waiting for `install-chaincode job`. 
Also for fabric version 2.x while addition of a new organization it should be waiting for the `approve-chaincode` job from the new organization

Reply:
should v.1.4.x be looking for `instantiate-chaincode job` instead?

Reply:
So in fabric v1.4.x we are invoking the chaincode after the addition of a new organization in already existing network where the chaincode is already instantiated, so we use invoke chaincode directly after installing it on the new peers and skipping the instantiate part as it has been already done on the channel.

Reply:
Ah I see, thank you.